frontend/src/components/Cart/CartSidebarItem.vue

```diff
@@ -19,8 +19,36 @@
 				font="text-md"
 			/>
 
-			<div class="flex flex-col justify-between w-full">
+			<div class="flex flex-col justify-between w-full relative">
 				<div class="text-sm font-semibold">{{ row.item_name }}</div>
+				<!-- Delete Menu Only Desktop Screens -->
+				<div class="cursor-pointer absolute -top-1 -right-1">
+					<Popover>
+						<template #target="{ togglePopover }">
+							<Button
+								:variant="'ghost'"
+								theme="gray"
+								size="sm"
+								@click.stop="togglePopover()"
+								class="w-[10px]"
+							>
+								<EllipsisVertical size="16px" />
+							</Button>
+						</template>
+						<template #body-main>
+							<Button
+								:variant="'ghost'"
+								theme="red"
+								size="sm"
+								label="Remove"
+								iconLeft="trash"
```

Comment:
use the same icons across the whole project. Use lucide's trash icon

Reply:
But ap nai filters component mai reset filter button kai sath button provided icon hi use kia tha to menai bhi wohi use karliya insted of library icon